src/commands/image-manipulation/triggered.js

```diff
@@ -10,14 +10,15 @@ module.exports = class Triggered extends Command {
       category: 'images',
       requirements: { canvasOnly: true },
       parameters: [{
-        type: 'user', full: true, required: false, acceptBot: true
+        type: 'image',
+        missingError: 'commands:morejpeg.missingImage'
       }]
     })
   }
 
-  async run ({ t, author, channel }, user = author) {
+  async run ({ t, author, channel }, image) {

```

Comment:
where is the user parameter :que:

Reply:
no need to, since it uses ImageParameter

Reply:
the imageparameter doesnt gets the author automatclly

Reply:
wait, lemme check

Reply:
nvm